docs/manifest/preprocessingdialog.md

```diff
@@ -0,0 +1,63 @@
+---
+title: PreProcessingDialog element in the manifest file (preview)
+description: The PreProcessingDialog element configures the pre-processing dialog of a spam reporting add-in in Outlook.
+ms.date: 07/14/2023
+ms.localizationpriority: medium
+---
+
+# PreProcessingDialog element (preview)
+
+Configures the pre-processing dialog of a spam reporting add-in in Outlook, so that users can provide additional information about the message they're reporting.
```

Comment:
```suggestion
Configures the preprocessing dialog of a spam reporting add-in in Outlook, so that users can provide additional information about the message they're reporting.
```